src/lib/Siteaccess/SiteaccessResolverInterface.php

```diff
@@ -39,6 +40,17 @@ public function getSiteAccessesListForLocation(
     ): array;
 
     /**
+     * @return \Ibexa\Core\MVC\Symfony\SiteAccess[]
+     */
+    public function getSiteAccessesListForContent(Content $content): array;
+
+    /**
+     * @return \Ibexa\Core\MVC\Symfony\SiteAccess[]
+     */
+    public function getSiteAccessesList(): array;
+
+    /**
+     * @deprecated use \Ibexa\AdminUi\Siteaccess\SiteaccessResolverInterface::getSiteAccessesList instead.
```

Comment:
```suggestion
     * @deprecated Deprecated since Ibexa DXP 4.5.0. Use { @see \Ibexa\AdminUi\Siteaccess\SiteaccessResolverInterface::getSiteAccessesList } instead.
```

+ trigger `E_DEPRECATED` error.